media/css/plugincheck/plugincheck.less

```diff
@@ -263,8 +263,100 @@ html[dir="rtl"] {
 #sec-plugin-uptodate {
     display: none;
 }
-.footer-newsletter-form {
-    padding: 40px 20px;
+#mozorg-newsletter-form,
+#newsletter-form {
+    width: 1300px / 2;
+    margin: 0 auto;
+    padding: 0 0 (@baseLine * 2) 0;
+    background: transparent;
+    border-bottom: none;
+    box-shadow: none;
+
+    h3 {
+        display: block;
+        float: none;
+        width: auto;
+        padding: (@baseLine * 2) 0;
+        color: @textColorSecondary;
+        .font-size(46px);
```

Comment:
This seems really big for an h3, even when centered as the main headline (see previous screenshot). I'd probably take this down to 32px.